.gitignore

```diff
@@ -86,6 +86,7 @@ celerybeat-schedule
 *.sage.py
 
 # Environments
+cisco_gnmi/
```

Comment:
Ideally we could use .venv and not need this.

Reply:
No problem, just trying to get a fancy venv name that goes with the project